crawl-ref/source/dat/des/variable/mini_monsters.des

```diff
@@ -2317,6 +2317,66 @@ cccc....cccc
 cccccccccccc
 ENDMAP
 
+NAME:   dreamdust_gnoll_farm
+TAGS:   no_monster_gen transparent
+DEPTH:  D:4-7
+COLOUR: pc = brown
+FTILE:  p = floor_mud
+KMONS:  p = fungus / nothing
+MONS:   gnoll ; scythe, hound
+NSUBST: . = 2=1 / 3=1. / 2 / 1=2. / .
+ITEM:   any potion no_pickup
+MARKER: ! = lua:fog_machine { cloud_type='flame', pow_min = 10, pow_max = 10, \
+                              delay = 10, size = 1, excl_rad = 0 }
+MAP
+..............
+.pppppp.ccccc.
+.pWWWWp.c.!dc.
+.pppppp.c...c.
+.pWWWWp.cc+cc.
+.pppppp.......
+..............
+ENDMAP
+
+NAME:   dreamdust_gnoll_shepherd
+TAGS:   no_monster_gen no_item_gen transparent
+DEPTH:  D:7-11, Lair
+KMONS:  1 = gnoll shaman name:gnoll_shepherd \
+            n_rpl n_des n_spe spells:none ; quarterstaff
+KMONS:  2 : yak / dream sheep
+NSUBST: . = 5=2 / .
+MAP
+.....
+..1..
+.....
+ENDMAP
+
+NAME:   dreamdust_looks_like_rain
+TAGS:   transparent no_monster_gen
```

Comment:
This one might want `patrolling` since there's a rare cloud type placement that will seem out of context if the player encounters the first time when the vault monster has wondered off.